### ISSUE TITLE
When I try to run the game, it doesn't support Direct11 and I can't run it.

### ISSUE THREAD

**Issue by remon-nomer66** [utmapp/UTM#6038]
**Configuration**  
* UTM Version: 4.4.5 (94)
* macOS Version: 14.2.1（23C71）
* Mac Chip (Intel, M1, ...): M1 Max

**Crash log**  
A D3D11-compatible GPU (Feature Level 11.0,Shader Model 5.0)is required
to run the engine.

**Comment by DUOLabs333** [comment]
Yes, this is a known issue --- I'm assuming you're either using Wine or using a Windows VM. ANGLE on mac
OS isn't full-featured to provide all of the needed features. Moving to Vulkan will probably help, but that is not a trivial endevour.
